README.md

```diff
@@ -440,6 +495,10 @@ $ helm repo update
 $ helm install --name minio --version 6.0.5 --set "accessKey=myaccesskey,secretKey=mysecretkey" minio/minio
 ```
 
+### Installing NATS
+
+TODO
```

Comment:
What should I do here? If we recommend a helm chart to install we should use it too for testing.

Reply:
You can simply link to here: https://nats-io.github.io/k8s
I don't think we need instruct the user how to install the Helm chart (I'm also thinking about removing these details for MongoDB and Minio, TBH). By the way, nats is an extremely simple service to run, no persistence, only a container running. I'm okay in testing it as we are doing now, with yamls applied with kubectl apply.